src/Cooker/index.js

```diff
@@ -28,5 +28,23 @@ export function Cooker(options) {
     tools.add(ft)
   }
 
-  return ft
+  return {
+    functionTree: ft,
+    run(name, ...args) {
```

Comment:
Nit picking, but this uglifies the API a bit in my opinion. What if we rather override the `run`? :-)

```js

const originalRun = ft.run
ft.run = () => {...}

return ft
```

So we avoid exporting two different things... or do you need access to basica run as well? If so, I would not call the method `run`, but rather `cook` :-)